scripts/fix_domain.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+# Copyright 2017, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# (c) 2017, Jean-Philippe Evrard <jean-philippe.evrard@rackspace.co.uk>
+
+## Shell Opts ----------------------------------------------------------------
+set -e -u -x
+set -o pipefail
+
+# Fix peroid sign on 'host_update' function
+sed -i 's/${RFCHOSTNAME}.${DOMAINNAME}/${RFCHOSTNAME}${DOMAINNAME}/g' /etc\
+\ansible\roles\openstack_hosts/templates/openstack-host-hostfile-setup.sh.j2
+
+# Fix 'openstack_host' in OSA-ops leap-upgrade
+sed -i 's/openstack.local/.openstack.local/g' /opt/rpc-leapfrog\
+/openstack-ansible-ops/leap-upgrades/upgrade-utilities-mitaka/playbooks\
+/old-hostname-compatibility-mitaka.yml
+sed -i 's/openstack.local/.openstack.local/g' /opt/rpc-leapfrog\
```

Comment:
We probably should do a check to see if these conditions exist before running a sed.  I could see this sed causing a problem if ran multiple times (which the script is supposed to be able to do) as I think it would add a dot per run.

Reply:
yeah, good ideas.